core/src/main/java/feast/core/service/SpecService.java

```diff
@@ -625,4 +634,110 @@ public UpdateStoreResponse updateStore(UpdateStoreRequest updateStoreRequest)
         .setStore(updateStoreRequest.getStore())
         .build();
   }
+
+  /**
+   * Applies the given Feature Table to the Feature Table registry. Creates the Feature Table if
+   * does not exist, otherwise updates the existing feature Table. Applies feature table in project
```

Comment:
Can we be consistent with casing.

Reply:
Made casing consistent throughout with `FeatureTable`. Let me know if you spot anymore inconsistent casing.